tools/renders/src/Renderer.js

```diff
@@ -75,7 +75,6 @@ class Renderer
             backgroundColor: 0xffffff,
             antialias: false,
             preserveDrawingBuffer: true,
-            roundPixels: true,
```

Comment:
This is the one place I couldn't think of how to replicate existing behaviour... since this test renderer renders both canvas and webgl at the same time...

Reply:
I’m not sure this is even critical for these tests. I’m thinking about removing these renders anyways in favor of the v5 render test suite that @Goodboydigital and I’ve have been building. Basically, don’t worry about it.